src/components/Square.js

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import './Square.css'
+import './Square.css';
 
 const Square = (props) => {
-  // For Wave 1 enable this 
-  //  Component to alert a parent 
-  //  component when it's clicked on.
+  const onSquareClicked = () => {
+    const updatedSquare = {
+      id: props.id,
+    };
+    props.onClickCallback(updatedSquare);
+  };
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
-}
+  return (
+    <button
+      disabled={props.gameStatus}
```

Comment:
Nice job passing this down from your app state, and using it to apply the game over styles. There are a few ways you could approach this, such as checking for whether the game is already over in the click handler to reject extra clicks, but this works as well.